### ISSUE TITLE
When force-exclude = true, file content vanishes

### ISSUE THREAD

**Issue by jankatins** [koxudaxi/ruff-pycharm-plugin#161]
**Describe the bug**

If I set this in pyproject.py:

```toml
force-exclude = true
exclude = [
    "package/subpackage/"
]
```

And then reformat a file in that subpackage (which calls `ruff --fix` in my case), the file content is empty afterwards.

**To Reproduce**
1. Install ruff plugin, configure it to run on reformat file
2. Configure ruff as above and open a file in that subpackage and add valid python
3. press reformat code
4. File is now empty, all content is gone

**Expected behavior**
Nothing should happen.

In any case, I would also expect that I do not need to configure that, but for that is https://github.com/koxudaxi/ruff-pycharm-plugin/issues/125 and the missing `--force-exclude`

**Environments (please complete the following information):**
 - IDE: Intelij, up to date
 - OS: mac osx up to date
 - Ruff Version: ruff 0.0.262
 - Plugin version: 0.0.13

**Comment by koxudaxi** [comment]
Version 0.0.14 has been published.

**Comment by pySilver** [comment]
I'm on 0.0.14 and whenever I attempt to reformat a directory all files within become empty.

**Comment by pySilver** [comment]
configuration:

```toml
[tool.ruff]
# D100: Missing docstring in public module
# D101: Missing docstring in public class
# D102: Missing docstring in public method
# D103: Missing docstring in public function
# D105: Missing docstring in magic method
# N806: Variable in function should be lowercase
# E501: Line too long
ignore = ["D100","D101","D102","D103","D105","N806","E501"]
exclude = [
 ".venv",
 ".tox",
 ".git",
 "*/migrations/*",
 "*/static/CACHE/*",
 "docs",
 "node_modules",
]
line-length = 119
select = [
 # Pyflakes
 "F",
 # Pycodestyle
 "E",
 "W",
 # isort
 "I001"
]

[tool.ruff.isort]
known-first-party = ["mybaze"]
known-third-party = ["willow","modelcluster","taggit","unidecode","bs4","pytz","PIL"]
section-order = ["future", "standard-library", "django", "wagtail", "third-party", "first-party", "local-folder"]

# Group all Django/Wagtail imports into a separate sections.
[tool.ruff.isort.sections]
"django" = ["django"]
"wagtail" = ["wagtail"]
```

**Comment by koxudaxi** [comment]
@pySilver 
> I'm on 0.0.14 and whenever I attempt to reformat a directory all files within become empty.

How did you reformat a directory in PyCharm?

**Comment by KMilhan** [comment]
Same here even without `force-exclude=true` option. All my `tests` (excluded) files are vanished when I reformat (individually, or per-directory) files.
Due to this, I turned of triggering ruff when reformating which I really wish to use in the main src edit.

**Comment by pySilver** [comment]
> @pySilver
> 
> > I'm on 0.0.14 and whenever I attempt to reformat a directory all files within become empty.
> 
> How did you reformat a directory in PyCharm?

Option+command+L or Right-click on folder and select "Format category" from context menu.

**Comment by thosil** [comment]
Hello, same problem here with this in pyproject.toml
```toml
extend-exclude = [
    "testing"
]
```

**Comment by koxudaxi** [comment]
Hi everyone, I have released the fixed version `0.0.24`
I hope the issue had been fixed by the changes.
